experimental/PropertyDDS/packages/property-binder/src/data_binder/dataBindingRegistry.ts

```diff
@@ -29,14 +31,14 @@ class DataBindingRegistry {
    * be created in the same order in which they have been registered. The same DataBindingObject may be registered
    * multiple times and there are no checks to prevent this
    *
-   * @param {string} bindingType The type of the representation.  (ex. 'VIEW', 'DRAW', 'UI', etc.)
-   * @param {string} typeid The id to use for this registration, usually the type id of the objects being represented
+   * @param bindingType The type of the representation.  (ex. 'VIEW', 'DRAW', 'UI', etc.)
```

Comment:
Just FYI. API-extractor will expect following format of parameter description( missing hyphen): 
```markdown
@param bindingType - The type of the representation. (ex. 'VIEW', 'DRAW', 'UI', etc.) 
```